### PR TITLE
Chapter 6 fixes

### PR DESCRIPTION
**Pull request by rhattersley**

### CODE REVIEW

**Comment by rhattersley** [comment]
Best left until Appendix H has been updated as the updated link to the example may not be valid.

File: ch06.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ element of an axis. This is particularly useful for discrete axes
 observational data from a number of observing stations, it may be
 convenient to provide the names of the stations as labels for the
 elements of the station dimension (<<time-series-data>>).
-      <<Example H.1.1>> illustrates another application for labels.
+      <<example-H.1.1>> illustrates another application for labels.
```
Comment (rhattersley) [inline ch06.adoc]: This will be wrong.

**Comment by marqh** [comment]
#23 merged, is this one good to go now?

**Comment by rhattersley** [comment]
> is this one good to go now?

Yes - I've rebased and adjusted the link so it's consistent.

**Pull request merged**